From bf4951029e5ffff944dbcb569392f41e4f13f7e3 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Sun, 25 Aug 2024 17:44:03 -0700
Subject: [PATCH] ==== Docs/damon: update GitHub repo URLs and
 maintainer-profile ====

Docs/damon: update GitHub repo URLs and maintainer-profile

Misc updates for DAMON documentation.

Replace GitHub URLS on DAMON documents for none-kernel parts DAMON repos
with new ones[1] via the first patch.  With following two patches,
wordsmith maitnainer-profile for better readability, and document the
Google clendsar for bi-weekly meetups, respectively.

[1] https://lore.kernel.org/20240813232158.83903-1-sj@kernel.org

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/tDG4zyMq | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/tDG4zyMq

diff --git a/damon_meta_changes/tDG4zyMq b/damon_meta_changes/tDG4zyMq
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.2

